config/jobs/kubernetes-sigs/cluster-api-provider-aws/cluster-api-provider-aws-presubmits.yaml

```diff
@@ -302,7 +302,7 @@ presubmits:
       preset-aws-credential: "true"
     spec:
       containers:
-        - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20221207-9bb38b7b14-master
```

Comment:
Just checking that this was intentional, right? We are no longer want to run master here?

Reply:
Yes this was intentional. I feel if other tests are running in 1.25, these tests should also run in the same, and should not use master. Wdyt?

Reply:
Yep, I agree.:)